skema/text_reading/text_reading/src/main/scala/org/ml4ai/skema/text_reading/grounding/MiraEmbeddingsGrounder.scala

```diff
@@ -107,23 +107,17 @@ object MiraEmbeddingsGrounder{
   }
 
 
-  def filterRelevantTerms(terms:IndexedSeq[GroundingConcept]): IndexedSeq[GroundingConcept] = {
-    
-    val relevantPrefixes = Set(
-      "vo",
-      "trans",
-      "apollosv",
-      "ncit",
-      "cemo",
-      "covoc"
-    )
-
-    terms filter {
-      term => {
-        val prefix = term.id.split(":").head
-        relevantPrefixes contains prefix
+  def filterRelevantTerms(terms:IndexedSeq[GroundingConcept], relevantNamespaces:Set[String]): IndexedSeq[GroundingConcept] = {
+    if(relevantNamespaces.nonEmpty){
+      terms filter {
+        term => {
+          val prefix = term.id.split(":").head
+          relevantNamespaces contains prefix
+        }
       }
     }
+    else
+      terms
```

Comment:
To be sure, if there are no relevantNamespaces, act as if there are all namespaces?  It sounds a little like relevantNamespacesis an Option[Set[String]] rather than a potentially empty set.